Projects/GLU/GLU01/GLU01-AWS100.md

```diff
@@ -0,0 +1,38 @@
+# GLU02-AWS100 - Introduction to SNS (Simple Notification Service)
+
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
+## Project's Author(s)
+
+- [Edward Allen Mercado](https://twitter.com/edwardmercado_)
+
+## Objectives
+
+### You need to complete the following:
+
+- Create a SNS Topic
+- Subscribe to a SNS Topic
+- Receive email notification from the SNS Topic created
+
+### You need to answer the following:
+
+- What is SNS?
+- How Pub/Sub notifications work?
+- What are the different endpoints that SNS can send notification?
+
```

Comment:
**Phrasing**
- How do Pub/Sub notifications work?
- What are the different endpoints that SNS can send notifications to?

**I'd add**
- What happens if a message in SNS can not be delivered?
- What's the difference between SNS and SQS?

Reply:
added them now on my latest commit

Reply:
The first two's grammar is still a bit lacking, can you please fix those missing "s" and "do"s?

Reply:
amended them, please review